recipe/add_version.patch

```diff
@@ -0,0 +1,6 @@
+--- /dev/null   2019-10-15 15:53:21.000000000 -0500
++++ .version    2019-10-15 15:53:05.719976400 -0500
+@@ -0,0 +1,3 @@
++VERSION=1.14
++RELEASE=1
++FULL=1.14
```

Comment:
This recipe has always been reporting `version 1.0` because 1. the source is not pulled from git repo 2; there is no `.version` file to report version.